.buildkite/pipelines/prod_sync_pipeline.yml

```
@@ -8,7 +8,20 @@ steps:
     depends_on: "start"
     commands:
       - cd .metrics/buildkite
+      - DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -y curl
       - curl -o ci_integrations -k https://github.com/platform-platform/monorepo/releases/download/ci_integrations-snapshot/ci_integrations_linux -L
       - chmod a+x ci_integrations
       - eval "echo \"$(sed 's/"/\\"/g' prod_integration_config.yml)\"" >> integration.yml
       - ./ci_integrations sync --config-file integration.yml --no-coverage --initial-sync-limit=2000 -v
+    plugins:
+      - docker-compose:
+          config: ".buildkite/compose-files/docker-compose.yml"
+          run: ubuntu
+          workdir: /monorepo
+          env:
+            - PROD_BUILDKITE_TOKEN
+            - PROD_PIPELINE_SLUG
+            - PROD_WEB_APP_USER_EMAIL
+            - PROD_WEB_APP_USER_PASSWORD
+            - PROD_CI_INTEGRATIONS_FIREBASE_API_KEY
+            - PROD_METRICS_PROJECT_ID
```

It seems like the `prod_integration_config.yml` requires more environment variables (please consider [this file](https://github.com/platform-platform/monorepo/blob/master/.metrics/buildkite/prod_integration_config.yml)):
```suggestion
          env:
            - PROD_BUILDKITE_TOKEN
            - PROD_PIPELINE_SLUG
            - PROD_ORGANIZATION_SLUG
            - PROD_FIREBASE_PROJECT_ID
            - PROD_WEB_APP_USER_EMAIL
            - PROD_WEB_APP_USER_PASSWORD
            - PROD_CI_INTEGRATIONS_FIREBASE_API_KEY
            - PROD_METRICS_PROJECT_ID
```